docs/zh/reference/information_schema/tables.md

```diff
@@ -0,0 +1,33 @@
+---
+displayed_sidebar: "Chinese"
+---
+
+# tables
+
+`tables` 提供有关表的信息。
+
+`tables` 提供以下字段：
+
+| **字段**        | **描述**                                                     |
+| --------------- | ------------------------------------------------------------ |
+| TABLE_CATALOG   | 表所属的 Catalog 名称。                                      |
+| TABLE_SCHEMA    | 表所属的数据库名称。                                         |
+| TABLE_NAME      | 表名。                                                       |
+| TABLE_TYPE      | 表的类型。 有效值：`BASE TABLE` 和 `VIEW`。                  |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dfa065b3-dfce-47ae-b456-d81bdbdc4720)
remove the space.